docs/sources/developers/kinds/composable/tempodataquery/schema-reference.md

```diff
@@ -0,0 +1,27 @@
+---
+keywords:
+  - grafana
+  - schema
+title: TempoDataQuery kind
+---
+> Both documentation generation and kinds schemas are in active development and subject to change without prior notice.
+
+# TempoDataQuery kind
+
+## Maturity: experimental
+## Version: 0.0
+
+## Properties
+
+| Property          | Type   | Required | Description                                                                                                          |
+|-------------------|--------|----------|----------------------------------------------------------------------------------------------------------------------|
+| `limit`           | number | No       | Defines the maximum number of traces that are returned from Tempo                                                    |
+| `maxDuration`     | string | No       | Define the maximum duration to select traces. Use duration format, for example: 1.2s, 100ms                          |
+| `minDuration`     | string | No       | Define the minimum duration to select traces. Use duration format, for example: 1.2s, 100ms                          |
+| `query`           | string | No       | TraceQL query or trace ID                                                                                            |
```

Comment:
JFYI `query` should be marked as required (not optional `?` in schema), but it doesn't b/c of https://github.com/grafana/schematization-and-as-code-project/issues/57

cc/ @undef1nd